00DIST

```diff
@@ -5295,11 +5295,14 @@ July 14, 2018
 		found
 		submitted by @emaste
 
-               Fix broken LSOF_CFLAGS_OVERRIDE.
-               Provided by Fabrice Fontaine in #172.
+		Fix broken LSOF_CFLAGS_OVERRIDE.
+		Provided by Fabrice Fontaine in #172.
 
-               Fix broken AR and RANLIB when cross-compiling.
-               Provided by Fabrice Fontaine in #194.
+		Fix broken AR and RANLIB when cross-compiling.
+		Provided by Fabrice Fontaine in #194.
+
+		Remove sysvlegacy function.
```

Comment:
Could you add `[linux]` as a prefix because this change is for the Linux dialect?
In addition, could you add `[linux]` to the header of the commit log like:
```
[linux]  dialects/linux/machine: remove sysvlegacy function 
```